configs/_base_/models/efficientnet_es.py

```diff
@@ -0,0 +1,16 @@
+# model settings
+model = dict(
+    type='ImageClassifier',
+    backbone=dict(
+        type='EfficientNet',
+        arch='es',
+        conv_cfg=dict(type='Conv2dAdaptivePadding'),
+        norm_cfg=dict(type='BN', eps=1e-3)),
```

Comment:
Since all configs use the same `conv_cfg` and `norm_cfg`, just use them as the default value.